Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -100,7 +100,7 @@ final class MongoCollectionTests: MongoSwiftTestCase {
     }
 
     func testAggregate() throws {
-        expect(Array(try self.coll.aggregate([["$project": ["_id": 0, "cat": 1]]])))
+        expect(Array(try self.coll.aggregate([["$project": ["_id": 0, "cat": 1]]]).all()))
```

Comment:
do you still need to wrap this in `Array(...)`, given that `all` returns an array?